backend/api/migrations/20210915232840_category_user_scope.sql

```diff
@@ -0,0 +1 @@
+alter table category add column user_scopes smallint[] not null default '{}';
```

Comment:
1. Migration should be dated so that it is run last. Other migrations have since been added to the repo since this was created locally
2. Missing a newline char at the end of file